### ISSUE TITLE
Not bypassing PHP

### ISSUE THREAD

**Issue by StrategyLLC** [maximejobin/rocket-nginx#70]
Currently, on my website https://strategynewmedia.com I am getting the following response headers:
X-Rocket-Nginx-Serving-Static: No
X-Rocket-Nginx-Reason: File Not Cached
X-Rocket-Nginx-File: /srv/strategynewmedia.com/wp-content/cache/wp-rocket/strategynewmedia.com//improve-websites-search-ranking//index-https.html

When I check for the file it is actually there so it the caching works fine. The problem is the directory is off. Wordpress is installed in a subdirectory named sb so the path should look like this: /srv/strategynewmedia.com/sb/...

When I move the include around in my configuration I get various paths. 

Mobile cache is off and I have not edited any of the rocket-nginx configurations other than turning debug on.

**Comment by haydenjames** [comment]
are you using cloudflare page rules?

**Comment by maximejobin** [comment]
@StrategyLLC There is a configuration named `wp_content_folder` in the ini file that can be changed for your specific case. You can change the value for: `sb/wp-content` and it should work (if I understand your situration correctly).

**Comment by StrategyLLC** [comment]
There are many different sites on this server using the configuration. How can I generate a config specific for each website?

**Comment by maximejobin** [comment]
In the ini file, create a new section that is derived from the default one like this:

    [myotherconfig : default]
    
    ; WP-Content folder
    ; Default value: wp-content
    wp_content_folder = hello/wp-content

Then, re-run the parser and you'll end up with 2 files
- default.conf
- myotherconfig.conf

Simply include the first configuration in one website and the second one in the other! You can repeat these steps for as many sites as you want.

**Comment by StrategyLLC** [comment]
Since this might be related to the same issue, I'm not closing the ticket yet. When the subdirectory is  added or in the case of another website jethq.com the headers are as follows for every inside page.
X-Rocket-Nginx-Serving-Static: No
X-Rocket-Nginx-Reason: Arguments Found
X-Rocket-Nginx-File: /srv/jethq.com/wp-content/cache/wp-rocket/jethq.com//aircraft-consulting//index-https.html

**Comment by maximejobin** [comment]
Arguments were found in your URL. That is why the cache is not served. Check the documentation about the reasons here:
https://github.com/maximejobin/rocket-nginx#debug

**Comment by StrategyLLC** [comment]
Right. I read all the documentation. This is happening on all pages whether there is a query string or not so I'm not sure if it has to do with pretty permalinks or something else.

**Comment by maximejobin** [comment]
What is the permalink structure? If it's the default one, it's totally normal! You should change it.

**Comment by StrategyLLC** [comment]
/%postname% is the permalink structure.

**Comment by maximejobin** [comment]
Is the `X-Rocket-Nginx-File` in the header the right one ? Otherwise, where is the cached file on your server ?

**Comment by StrategyLLC** [comment]
Finally. I got everything working. I needed a new configuration file to use for my sites in a subdirectory. I got all that figured out; however, pages were still not bypassing. The whole problem with this was I had the included file inside the location block for PHP-FPM. I moved it to the top of my server block and everything works. This piece of software is insane. It got my sites from 1-2 seconds down to .3-.7 seconds.
